src/MainWindow.vala

```diff
@@ -684,7 +684,14 @@ namespace Terminal {
 
         private void on_tab_moved (Granite.Widgets.Tab tab, int x, int y) {
             Idle.add (() => {
-                var new_window = app.new_window_with_coords (x, y, false);
+                var new_window = new MainWindow.with_coords (
+                    (Terminal.Application) GLib.Application.get_default (),
```

Comment:
Can't you still use `app` here though?

Reply:
No because it's another class and `app` is an instance member

Reply:
Oops nevermind, I'm dumb. I was looking at something else. Fixed